diff --git a/src/test/java/org/openhab/automation/jrule/rules/value/JRuleIncreaseDecreaseValueTest.java b/src/test/java/org/openhab/automation/jrule/rules/value/JRuleIncreaseDecreaseValueTest.java
index 27e6220..e628aa1 100644
--- a/src/test/java/org/openhab/automation/jrule/rules/value/JRuleIncreaseDecreaseValueTest.java
+++ b/src/test/java/org/openhab/automation/jrule/rules/value/JRuleIncreaseDecreaseValueTest.java
@@ -40,8 +40,8 @@ class JRuleIncreaseDecreaseValueTest {
     @Test
     void fromOhCommand() {
         Assertions.assertEquals(JRuleIncreaseDecreaseValue.INCREASE,
-                JRuleEventHandler.get().toValue(IncreaseDecreaseType.INCREASE));
+                JRuleEventHandler.toValue(IncreaseDecreaseType.INCREASE));
         Assertions.assertEquals(JRuleIncreaseDecreaseValue.DECREASE,
-                JRuleEventHandler.get().toValue(IncreaseDecreaseType.DECREASE));
+                JRuleEventHandler.toValue(IncreaseDecreaseType.DECREASE));
     }
 }
